packages/types/Calendar.d.ts

```diff
@@ -93,6 +93,7 @@ export interface CalendarEvent {
   cancellationReason?: string | null;
   rejectionReason?: string | null;
   hideCalendarNotes?: boolean;
+  isOrganizer: boolean;
```

Comment:
Is the `isOrganizer` property required? 

Because this is causing a lot of errors when creating a `CalendarEvent`. Take a look into the [Check types workflow](https://github.com/kommitters/cal.com/runs/5991702217?check_suite_focus=true)

Reply:
Thanks for making me realize that the `isOrganizer` property is not required, I'll change it to optional.